tools/distpartitioning/utils.py

```diff
@@ -8,29 +8,38 @@
 import pyarrow
 from pyarrow import csv
 
-def read_partitions_file(part_file):
+def read_ntype_partition_files(schema_map, input_dir):
     """
-    Utility method to read metis partitions, which is the output of 
-    pm_dglpart2
+    Utility method to read the partition id mapping for each node.
+    For each node type, there will be an file, in the input directory argument
+    containing the partition id mapping for a given nodeid. 
```

Comment:
if you return a single numpy, you basically assume the returned partition id array will be used for the homogenized graph. if so, you need to explain it in the doc string.

Reply:
Yes. This as per the design we all agreed upon. I will elaborate the docstring to make clear about this issue.